core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -332,6 +332,7 @@ object Defaults {
   val LiLogCleanerFineGrainedLockEnabled = true
   val LiDropCorruptedFilesEnabled = false
   val LiConsumerFetchSampleRatio = 0.01
+  val LiZookeeperPaginationEnable = false
```

Comment:
Are we planning to enable this only in clusters that have topic-name-length larger than some threshold? Or do we want to enable it by default? Any performance degradation for calling paginated one?